hack/collector-longhorn

```diff
@@ -17,6 +17,33 @@ function collect_multipath_config(){
     fi
 }
 
+function collect_processes_info(){
+    collect_pids_status
+    collect_processes_status
+}
+
+function collect_pids_status(){
+    output="./proc_pids_status"
+    : > "$output"   # truncate/clear the file first
+
+    for pid in ${HOST_PATH}/proc/[0-9]*; do
+        status="$pid/status"
+
+        if [ -f "$status" ]; then
+            echo "===== PID $(basename "$pid") =====" >> "$output"
+            cat "$status" >> "$output"
+            echo >> "$output" # add a newline for better readability
+        fi
+    done
```

Comment:
Suggest using `ps aux` or with other options.

Reply:
Why do we need `collect_pids_status`?
`collect_processes_status` should be enough.